### PR TITLE
Patch documentation site

### PR DESCRIPTION
**Pull request by VeckoTheGecko**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -10,6 +9,7 @@ dependencies:
   - opensimplex == 0.4.5
   - numpy >=1, < 2
   - pydantic >=2, <3
+  - pip
```
Comment (VeckoTheGecko) [inline environment.yml:12]: Readthedocs was complaining about not having an explicit pip dependency in its logs.

File: .readthedocs.yaml [hunk 2]
```diff
@@ -9,7 +9,8 @@ build:
     python: mambaforge-22.9
   jobs:
     pre_build:
-      - sphinx-build -b linkcheck docs/ _build/linkcheck
+      - pip install .
+      # - sphinx-build -b linkcheck docs/ _build/linkcheck
```
Comment (VeckoTheGecko) [inline .readthedocs.yaml:13]: Disabling until Conda is published etc. and all the links are working properly.

File: README.md [hunk 3]
```diff
@@ -96,9 +96,7 @@ Options:
 
 ```
 
-For examples, see LINK_TO_TURORIALS.
-
-<!-- TODO: Link to tutorials -->
+For examples, see [the tutorials section of our documentation](https://virtualship.readthedocs.io/en/latest/tutorials/index.html).
```
Comment (erikvansebille) [inline README.md:99]: Do we also want to enable virtualship.oceanparcels.org, and use that url here already?

Comment (erikvansebille) [reply README.md]: OK, also done on my side
Screenshot 2024-10-08 at 13 48 46

May take a while for the DNS to become active?

Comment (VeckoTheGecko) [reply README.md]: Looks good. i think we're good to merge

**Review by erikvansebille — APPROVED** [review]

**Pull request merged**